index.html

```diff
@@ -55,12 +54,6 @@ <h2>Jobs & Projects</h2>
             <a href="/jobs">See all jobs</a>
         </div>
     </div>
-    <div class="row">
-        <div class="col-sm-12 activity">
-            <h2>Activity</h2>
-            <ul id="activity-feed" class="activity"></ul>
-        </div>
-    </div>
```

Comment:
Same here, fine to remove as it’s the part which just showed Github activity, which is not our main thing.